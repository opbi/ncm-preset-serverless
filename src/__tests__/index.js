import join from 'index';

describe('join', () => {
  it('works if input is string', () => {
    const result = join('hello', 'world');
    expect(result).toBe('hello world');
  });

  it('would fail if input is not string', () => {
    const corrupted = () => join(2, 'yes');
    expect(corrupted).toThrowErrorMatchingSnapshot();
  });
});
