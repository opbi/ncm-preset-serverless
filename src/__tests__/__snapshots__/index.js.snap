// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`join would fail if input is not string 1`] = `"function validation failed"`;
